### PR TITLE
fix: do not hardcode npx in shadcn-ui:add executor

### LINKED ISSUE #245: `shadcn-ui` not working with pnpm
`npx` is hard-coded in `shadcn-ui` package which doesn't properly work with `pnpm`

### PR DESCRIPTION
**Pull request by gurisko**
Fixes #245

### CODE REVIEW

File: packages/shadcn-ui/src/executors/add/add.impl.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export async function addExecutor(
   const { root } = context.workspace.projects[context.projectName]
 
   return execCommand(buildCommand([
-    'npx shadcn-ui@latest add',
+    `${getPackageManagerCommand().dlx} shadcn-ui@latest add`,
```
Comment (TriPSs) [inline packages/shadcn-ui/src/executors/add/add.impl.ts:16]: Could you change this to use `execPackageManagerCommand` from `@nx-extend/core`.

Comment (TriPSs) [reply packages/shadcn-ui/src/executors/add/add.impl.ts:16]: See for example the [firebase hosting one](https://github.com/TriPSs/nx-extend/blob/master/packages/firebase-hosting/src/executors/deploy/deploy.impl.ts#L22)

Comment (gurisko) [reply packages/shadcn-ui/src/executors/add/add.impl.ts:16]: That won't work. `execPackageManagerCommand` is using:
```
export function getPackageManagerExecCommand() {
  return getPackageManagerCommand(detectPackageManager()).exec
}
```

which will then run:
```
pnpm exec shadcn-ui@latest add {{component}} --path=src --cwd={{root}}
```
`pnpm` alternative to `npx` is `pnpm dlx`, not `pnpm exec` ([source](https://pnpm.io/feature-comparison)).

Comment (gurisko) [reply packages/shadcn-ui/src/executors/add/add.impl.ts:16]: Do you think we should create `dlxPackageManagerCommand()` in `@nx-extend/core` then?

Comment (TriPSs) [reply packages/shadcn-ui/src/executors/add/add.impl.ts:16]: Cool didn't now that something like `npx` existed in the other package managers. Let's update update `getPackageManagerExecCommand` indeed. Good one.

**Review by TriPSs — APPROVED** [review]

**Pull request merged**